x-pack/plugins/stack_alerts/server/alert_types/index_threshold/alert_type_params.test.ts

```diff
@@ -121,24 +127,24 @@ export function runTests(schema: ObjectType, defaultTypeParams: Record<string, u
 
       params.index = 42;
       expect(onValidate()).toThrowErrorMatchingInlineSnapshot(`
-"[index]: types that failed validation:
-- [index.0]: expected value of type [string] but got [number]
-- [index.1]: expected value of type [array] but got [number]"
-`);
+        "[index]: types that failed validation:
+        - [index.0]: expected value of type [string] but got [number]
+        - [index.1]: expected value of type [array] but got [number]"
+      `);
```

Comment:
I'm curious, not a blocker, what change causes this different snapshots?

Reply:
I guess schema change affected this. There is no direct cause of this.

Reply:
@afharo you might know why this snapshot changed?

Reply:
thx,anyway, approving it

Reply:
The only change is the spacing... My assumption is that `jest` added it when editing the file to write the inline snapshot for the previous test.

I just tested it on `main`, just duplicate line 129 without providing the inline snapshot and run the `jest` tests. When writing the inline snapshot it "fixes" the spaces for the rest.